INFO: localdir at /scratch/98400.gcrepaldi
INFO: your job will run on local system.
Using GPU: cuda:0
Using num thread = 2
Traceback (most recent call last):
  File "setup.py", line 138, in <module>
    df_prices = pd.read_csv("./hotel_prices.csv")
  File "/home/gcrepaldi/touringrec/env/lib/python3.5/site-packages/pandas/io/parsers.py", line 702, in parser_f
    return _read(filepath_or_buffer, kwds)
  File "/home/gcrepaldi/touringrec/env/lib/python3.5/site-packages/pandas/io/parsers.py", line 429, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
  File "/home/gcrepaldi/touringrec/env/lib/python3.5/site-packages/pandas/io/parsers.py", line 895, in __init__
    self._make_engine(self.engine)
  File "/home/gcrepaldi/touringrec/env/lib/python3.5/site-packages/pandas/io/parsers.py", line 1122, in _make_engine
    self._engine = CParserWrapper(self.f, **self.options)
  File "/home/gcrepaldi/touringrec/env/lib/python3.5/site-packages/pandas/io/parsers.py", line 1853, in __init__
    self._reader = parsers.TextReader(src, **kwds)
  File "pandas/_libs/parsers.pyx", line 387, in pandas._libs.parsers.TextReader.__cinit__
  File "pandas/_libs/parsers.pyx", line 705, in pandas._libs.parsers.TextReader._setup_parser_source
FileNotFoundError: [Errno 2] File b'./hotel_prices.csv' does not exist: b'./hotel_prices.csv'
